File: C/Colleen.c
#include <stdio.h>

/*
	Outside comment
*/

void additional_function()
{
	char *srcs = "#include <stdio.h>%c%c/*%c	Outside comment%c*/%c%cvoid additional_function()%c{%c	char *srcs = %c%s%c;%c	printf(srcs, 10, 10, 10, 10, 10, 10, 10, 10, 34, srcs, 34, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10);%c}%c%cint	main()%c{%c	/*%c		Inside comment%c	*/%c	additional_function();%c	return (0);%c}";
	printf(srcs, 10, 10, 10, 10, 10, 10, 10, 10, 34, srcs, 34, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10);
}

int	main()
{
	/*
		Inside comment
	*/
	additional_function();
	return (0);
}

File: PYTHON/Colleen.py
# Outside comment

def additional_function():
	source_code = '# Outside comment\n\ndef additional_function():\n\tsource_code = %r\n\tprint(source_code %% source_code)\n\ndef main():\n\t# Inside comment\n\tadditional_function()\n\nmain()'
	print(source_code % source_code)

def main():
	# Inside comment
	additional_function()

main()
